settings/blt.settings.php

```diff
@@ -302,8 +302,16 @@
   if (file_exists(DRUPAL_ROOT . "/sites/settings/local.settings.php")) {
     require DRUPAL_ROOT . "/sites/settings/local.settings.php";
   }
-  // Load local settings for given single.
+  // Since Drupal 8.6, the file is recommended to be named settings.local.php.
+  if (file_exists(DRUPAL_ROOT . "/sites/settings/settings.local.php")) {
+    require DRUPAL_ROOT . "/sites/settings/settings.local.php";
+  }
+  // Load local settings for given single site.
   if (file_exists(DRUPAL_ROOT . "/sites/$site_dir/settings/local.settings.php")) {
     require DRUPAL_ROOT . "/sites/$site_dir/settings/local.settings.php";
```

Comment:
Could we trigger some kind of warning if using the old name maybe? Something like...
```php
@trigger_error('Drupal has changed the recommended local settings filename convention from local.settings.php to settings.local.php. Please update your file, as it will no longer be included by BLT in 11.x.', E_USER_DEPRECATED);
```